src/demos/datepicker/fuzzy-datepicker-demo.component.html

```diff
@@ -0,0 +1,133 @@
+<h2
+  class="sky-section-heading"
+>
+  Standard fuzzy datepicker
+</h2>
+
+<div
+  style="max-width: 50%"
+>
+  <sky-datepicker>
+    <input
+      skyFuzzyDatepickerInput
+      [futureDisabled]="futureDisabled"
+      [dateFormat]="dateFormat"
+      [disabled]="disabled"
+      [maxDate]="maxDate"
+      [minDate]="minDate"
+      [skyFuzzyDatepickerInput]
+      [skyDatepickerNoValidate]="noValidate"
+      [startingDay]="startingDay"
+      [yearRequired]="yearRequired"
+      [(ngModel)]="selectedDate"
+      #date="ngModel"
+    />
+  </sky-datepicker>
+
+  <p *ngIf="date.errors && date.errors.skyFuzzyDate && (date.dirty || date.touched)"
+    class="sky-error-label"
+  >
+    <span *ngIf="date.errors.skyFuzzyDate?.invalid">
+      Please enter a valid date.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.yearRequired">
+      Please select a year.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.futureDisabled">
+      Date cannot be in the future.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.maxDate">
+      The current date is beyond the maximum allowed date.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.minDate">
+      The current date is before the minimum allowed date.
+    </span>
+  </p>
+</div>
+
+<p>
+  Selected fuzzy date is {{ selectedDateForDisplay }}
```

Comment:
This is currently a sentence fragment with no punctuation at the end. I'd either delete "is" and add a colon after "date:" to make it more like a key-value pair, or else I'd add "The" to the start, lowercase "selected", and add a period to the end to make this a complete sentence.

Reply:
Done.

Reply:
Same question here about formatting the date. I should have thought of this yesterday. Sorry. ... It may not be an issue, but it seems worth asking the question. Maybe we need to get Todd to weigh in on whether this is worth addressing?

Reply:
This part is meant to be for the developers. It's showing you what the object looks like. It's not intended for end users to see. Should we make that more clear?

Reply:
I can't think of a way to make that more clear. Let's leave it as is for now, and I'll circle back if I can think of a better way to make this distinction.